### ISSUE TITLE
Progress log of training with scale 4x?

### ISSUE THREAD

**Issue by GabrielLin** [jiny2001/deeply-recursive-cnn-tf#3]
Could you please show you progress log of training with scale 4x? I used my custom domain images for training and want to compare with your result? Thanks.

**Comment by jiny2001** [comment]
OK. I will try training for x4 tonight. It may take whole day but I will post that when it have done. :)

**Comment by jiny2001** [comment]
As I posted on another topic, 4x with yang91+general100 training dataset leads to Set5:30.91 Set14:27.76 BSD100:27.09. It took almost 2days to process 100+ epochs. I'll be happy if you share your custom domain results when you get it. ;)

**Comment by GabrielLin** [comment]
I trained with my custom domain 200 images and validation set is also my custom image. The training terminated at 68 epoch with best 31.75. I got Set5 29.0, BSD100 26.1. The results are a little different when testing many time.

**Comment by GabrielLin** [comment]
I cannot test with set14. It seems that the shape of img_02 or img_03 is not 3. So it went to another banch of if len(image.shape) == 3 and image.shape[2] == 3 in function resize_image_by_pil_bicubic and got error.

**Comment by jiny2001** [comment]
I tried using Set14 for both training and test, and both looks working. Which command are you actually trying? For ex, I tried "python3 main.py --training_set Set14 --dataset set14" and there was no error like that.

**Comment by jiny2001** [comment]
BTW, thank you for sharing your results! Yes the result slightly differs because of the random initial values. I found Yang91+BSD200 is good for training.

**Comment by GabrielLin** [comment]
OOh, it it my fault. My modified some and it lead to the error of build grayscale image. It is OK now. My set14 PSNR is 26.4. Thanks.

**Comment by qingchuanhuajuan** [comment]
Hello ! Which is your X4 model? I changed the scale to 4 and the result was very poor.
